.travis.yml

```diff
@@ -2,7 +2,7 @@ version: "~> 1.0"
 env:
     global:
         - CI: true
-        - NODE_VERSION: 8.16.2
+        - NODE_VERSION: 14.15.4
```

Comment:
@brandongregoryscott probably need to bring over your list of commands for regression testing being this is changing the node major version. What are your thoughts? Happy to team up on QA'ing them like we did for the TS port

Reply:
Yeah, I've been thinking about opening up an issue to bump the node version requirements now that we've made the plunge as a team - I think that's the only thing preventing us from using a newer version of `ts-jest` and removing those warnings. 

@dylanjustice If the node version bump isn't required for this feature, could we revert this and capture this work in another issue?